src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -4893,6 +4893,45 @@ paths:
       summary: Get Monitor Details
       tags:
       - Monitor
+  /api/motd:
+    get:
+      parameters: []
+      responses:
+        '200':
+          content:
+            application/vnd.ceph.api.v1.0+json:
```

Comment:
As this endpoint is unlikely to be used for managing Ceph via REST, I'm wondering if there's an easy way to hide it/disable it (including the versioning, which is an overkill if this is not meant for RESTful usage).

Maybe via the UI API controller? @avanthakkar what's your take on this?

Reply:
I'm fine with that. If possible i would like to get rid of the API documentation too, because this endpoint is mostly used by the UI and does not need docs IMO.

Reply:
Done